README.rst

```diff
@@ -62,6 +62,9 @@ You can pass many options to the configure script; run ``./configure --help``
 to find out more.  On macOS and Cygwin, the executable is called ``python.exe``;
 elsewhere it's just ``python``.
 
+If you are running on Mac with the latest updates installed, make sure to install
+openSSL or some other SSL software along with Homebrew.
+
```

Comment:
There are other options besides Homebrew.  And other modules are needed.  Since this is somewhat of a moving target, suggest just referring to the macOS section of the Python Developer's Guide for now:  https://devguide.python.org/setup/#macos-and-os-x